lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
If you recall, my idea before was to return an Ruby object (call it KnackhqObject) that would have a name, multiple fields, and records.

For example:
```Ruby
Knackhq::Objects.new(app_id, app_key)
Returns:

[KnackhqObject(:knack_id => '1234',  :fields => {field_1=>{:feild_info....},:field_2=>{..}...}, :records=>[...], :total_records => 1000)....]
```

I think we decided against it for some reason. It would produce a bit more complexity but would return a model rather than a collection of methods. Honestly, It would make a good 1.0 feature set but may not be good just now.